pytorch-job/OWNERS

```diff
@@ -1,4 +1,5 @@
 approvers:
   - andreyvelich
```

Comment:
@andreyvelich @gaocegege @johnugeorge please LGTM

Reply:
/LGTM

Thanks for your contribution! :tada: :+1: